src/main/java/com/squareup/javapoet/CodeBlock.java

```diff
@@ -144,7 +147,7 @@ public Builder endControlFlow(String controlFlow, Object... args) {
       return this;
     }
 
-    public Builder statement(String format, Object... args) {
+    public Builder addStatement(String format, Object... args) {
```

Comment:
This and `add` need Javadoc to clarify their behavior and difference. Future PR.